klump-wc-payment.php

```diff
@@ -13,43 +13,46 @@
  * Plugin Name:       Klump WC Buy Now, Pay Later
  * Plugin URI:        https://useklump.com/
  * Description:       Buy Now, Pay Later (BNPL) plugin for Klump.
- * Version:           1.1.0
- * Requires at least: 5.2
- * Requires PHP:      7.2
- * WC requires at least: 5.0
- * WC tested up to: 6.3.1
+ * Version:           1.2.1
+ * WC requires at least: 7.0
+ * WC tested up to:   8.9.3
  * Author:            Klump Developers
  * Author URI:        https://useklump.com/developers
  * Text Domain:       klp-payments
  * License:           GPL v2 or later
  * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-if (!defined('ABSPATH')) {
+if ( ! defined('ABSPATH')) {
     exit;
 }
 
 define('KLP_WC_PLUGIN_FILE', __FILE__);
 define('KLP_WC_SDK_URL', 'https://js.useklump.com/klump.js');
-define('KLP_WC_SDK_VERIFICATION_URL', 'https://api.useklump.com/v1/transactions/');
+define('KLP_WC_SDK_VERIFICATION_URL', 'https://staging-api.useklump.com/v1/transactions/');
```

Comment:
This has been updated to staging. 
```suggestion
define('KLP_WC_SDK_VERIFICATION_URL', 'https://api.useklump.com/v1/transactions/');
```